eva_sub_cli/validators/docker_validator.py

```diff
@@ -67,18 +67,23 @@ def run_docker_validator(self):
 
             # copy all required files to container (mapping file, vcf and fasta)
             self.copy_files_to_container()
+            # create the output directory
+            self._run_quiet_command(
+                "Create output directory in container",
+                f"{self.docker_path} exec {self.container_name} mkdir -p {container_validation_dir}/{container_validation_output_dir}"
+            )
 
             docker_cmd = self.get_docker_validation_cmd()
             # start validation
-            # FIXME: If nextflow fails in the docker exec still exit with error code 0
```

Comment:
Is this actually fixed ? 
Nextflow is still run within docker and docker would still return 0 exit code

Reply:
I thought so, I tested explicitly raising an exception from `xlsx2json.py` and got this:
```
(venv) april@boudica:~/projects/tasks/subcli$ eva-sub-cli.py --submission_dir /home/april/projects/tasks/subcli --vcf_files /home/april/projects/tasks/subcli/RambouilletSheepGeno.vcf --assembly_fasta /home/april/projects/tasks/subcli/GCA_016772045.1.fa --metadata_xlsx /home/april/projects/tasks/subcli/EVA_Submission_RambouilletSheepGeno.xlsx --tasks VALIDATE --executor DOCKER
[2024-Apr-30 13:30:13][DockerValidator][INFO] Start validation
[2024-Apr-30 13:30:13][eva_sub_cli.validators.docker_validator][INFO] Container (eva-sub-cli.v0.0.1.dev6) is running
[2024-Apr-30 13:30:22][ebi_eva_common_pyutils.command_utils][ERROR] Run Validation using Nextflow failed! Refer to the error messages for details.
[2024-Apr-30 13:30:22][eva_sub_cli.validators.docker_validator][ERROR] Command 'docker exec eva-sub-cli.v0.0.1.dev6 nextflow run eva_sub_cli/nextflow/validation.nf --base_dir /opt/vcf_validation --vcf_files_mapping /home/april/projects/tasks/subcli/vcf_mapping_file.csv --metadata_xlsx /home/april/projects/tasks/subcli/EVA_Submission_RambouilletSheepGeno.xlsx --output_dir vcf_validation_output' returned non-zero exit status 1.
[2024-Apr-30 13:30:22][DockerValidator][INFO] Create report
[2024-Apr-30 13:30:22][DockerValidator][INFO] View the validation report in your browser: /home/april/projects/tasks/subcli/validation_output/report.html
(venv) april@boudica:~/projects/tasks/subcli$
```